js/views/userPageVw.js

```diff
@@ -381,6 +381,8 @@ module.exports = Backbone.View.extend({
       processData: false,
       data: formData,
       success: function(data) {
+        var errorModal = $('.js-messageModal');
+        errorModal.removeClass('hide');
```

Comment:
Moving the declaration here is fine, but the errorModal should only be unhidden when one of the error conditions is met.